docs/components/configurationPanel.md

```diff
@@ -0,0 +1,224 @@
+triggered by the [createMenu](#configurationpanelcreatemenu) invoker.
+
+### `{configurationPanel}.handleSwitching()`
+
+- Returns: Nothing.
+
+Listens to the changes in the "Input Name" dropdown menu and switches to the configuration menu for the selected gamepad
+input.
+
+### `{configurationPanel}.modifyActionDropdownMenu()`
+
+- Returns: Nothing.
+
+Displays or hides the "Speed Factor" input field and "Open a new tab/window in background" or "Invert Action" checkbox
+by calling the [changeConfigMenuOptions](#configurationpanelchangeconfigmenuoptionsdropdownmenu) invoker. The
+[createMenu](#configurationpanelcreatemenu) invoker triggers this method after the configuration option DOM elements are
+created.
+
+### `{configurationPanel}.listenActionDropdownChanges()`
+
+- Returns: Nothing.
+
+Listens to the changes in the "Action" dropdown of all the gamepad inputs' configuration menu and triggers
+[changeConfigMenuOptions](#configurationpanelchangeconfigmenuoptionsdropdownmenu) as a callback.
+
+### `{configurationPanel}.changeConfigMenuOptions(dropdownMenu)`
+
+- `dropdownMenu {Object}` The "Action" dropdown for a gamepad input.
+- Returns: Nothing.
+
+Displays or hides the "Speed Factor" input box and "Open a new tab/window in background" or "Invert Action" checkbox
+according to the value of the selected action in the corresponding "Action" dropdown.
+
+### `{configurationPanel}.setAllToNoneListener()`
+
+- Returns: Nothing.
+
+Listens to the click event on "Set All to None" button and sets the value of the "Action" dropdown for all the gamepad
+inputs to **None**.
+
+### `{configurationPanel}.setToDefaultListener()`
+
+- Returns: Nothing.
+
+Listens to the click event on "Restore Defaults" button and sets all the gamepad input configuration options to their
+default values.
+
+### `{configurationPanel}.discardChangesListener()`
+
+- Returns: Nothing.
+
+Listens to the click event on "Discard Changes" button and discards the unsaved changes to all the gamepad input
+configuration options.
+
+### `{configurationPanel}.saveChangesListener()`
+
+- Returns: Nothing.
+
+Listens to the click event on "Save Changes" button and saves the value of all the configuration options in Chrome's
+storage to be used by the [inputMapper](inputMapper.md) component.
+
+### `{configurationPanel}.toggleSaveAndDiscardButtons()`
+
+- Returns: Nothing.
+
+Toggles the state of "Discard Changes" and "Save Changes" button when the value of any configuration option on the panel
+is changed. The new value of the configuration options are compared to the last saved values, and if those are
+unchanged, then the buttons will be disabled. Otherwise, the buttons are enabled.
+
+### `{configurationPanel}.storeUnsavedChanges()`
+
+- Returns: Nothing.
+
+Stores all the unsaved changes in Chrome's storage when the value of any configuration option is changed. These values
```

Comment:
Please link to the Chrome storage API docs at least the first time you mention it on a page.